internal/envoy/versions.go

```diff
@@ -22,17 +22,16 @@ import (
 	"runtime"
 	"sort"
 
-	"github.com/tetratelabs/getenvoy/internal/transport"
 	"github.com/tetratelabs/getenvoy/internal/version"
 )
 
 // GetEnvoyVersions returns a version map from a remote URL. eg globals.DefaultEnvoyVersionsURL.
 func GetEnvoyVersions(envoyVersionsURL string) (version.EnvoyVersions, error) {
 	result := version.EnvoyVersions{}
 	// #nosec => This is by design, users can call out to wherever they like!
-	resp, err := transport.Get(envoyVersionsURL)
-	if err != nil {
-		return result, err
+	resp, e := httpGet(envoyVersionsURL)
+	if e != nil {
+		return result, e
```

Comment:
why not use `err` ? 😄

Reply:
switched everything to this form so no more inconsistency!